pkg/sql/contention/event_store_test.go

```diff
@@ -249,6 +254,114 @@ func BenchmarkEventStoreIntake(b *testing.B) {
 	}
 }
 
+func TestLogResolvedEvents(t *testing.T) {
+	ctx := context.Background()
+
+	// Set up log interception to capture structured AggregatedContentionInfo events
+	spy := logtestutils.NewStructuredLogSpy(
+		t,
+		[]logpb.Channel{logpb.Channel_SQL_EXEC},
+		[]string{"aggregated_contention_info"},
+		logtestutils.FromLogEntry[eventpb.AggregatedContentionInfo],
+	)
+	cleanup := log.InterceptWith(ctx, spy)
+	defer cleanup()
+
+	fingerprintIDToHex := func(id uint64) string {
+		return "‹\\x" + sqlstatsutil.EncodeStmtFingerprintIDToString(appstatspb.StmtFingerprintID(id)) + "›"
```

Comment:
Once you addd the `nonsensitive` tag to the protobuf you wont need these redaction tags anymore